### PR TITLE
Link the copyright statement in the footer to the ACF page

### LINKED ISSUE #389: Link the copyright statement in the footer to the ACF page
**Is your feature request related to a problem? Please describe.**
Currently there is no link provided to the `© 2022  Computer Fraternity, SACK` in the footer

**Describe the solution you'd like**
Add a link to redirect to the acf page

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by YoshithaRathnayake**
## Purpose

The purpose of this PR is to fix #389 

## Goals

Link the copyright statement in the footer to the ACF page

## Approach

I linked the copyright statement in the footer to the ACF page

### Screenshots

image

  
### Preview Link

https://pr-393-sack-site-acf.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [ ] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: _includes/static/footer.html [hunk 1]
```diff
@@ -14,14 +14,16 @@
...
-          &nbsp;Computer Fraternity, SACK
+            &nbsp;Computer Fraternity, SACK
+          </a>
         </div>
       </div>
     </div>
   </div>
 </footer>
-<!-- //Footer -->
+<!-- //Footer -->
```
Comment (Piumal1999) [inline _includes/static/footer.html:29]: Add a new line here

Comment (Piumal1999) [reply _includes/static/footer.html:29]: Also add a screenshot to the PR description

Comment (YoshithaRathnayake) [reply _includes/static/footer.html]: Done

**Comment by YoshithaRathnayake** [comment]
@Piumal1999

**Comment by YoshithaRathnayake** [comment]
@Piumal1999 I update the screenshot

**Review by Piumal1999 — APPROVED** [review]
Good job @YoshithaRathnayake 
Thank you

**Pull request merged**